buildpack.md

```diff
@@ -744,31 +749,39 @@ The stack ID:
 
 The stack `build-images` and `run-images` are suggested sources of images for platforms that are unaware of the stack ID. Buildpack authors MUST ensure that these images include all mixins specified in `mixins`.
 
-### launch.toml (TOML)
+### app.toml (TOML)
 
 ```toml
 [[processes]]
 type = "<process type>"
 command = "<command>"
+
+[[slices]]
+paths = ["<app sub-path glob>"]
 ```
 
-Buildpacks MUST specify:
+The buildpack MAY specify any number of processes or slices.
+
+For each process, the buildpack MUST specify:
 
-- A unique process type for each entry within a `launch.toml` file.
+- A unique process type for each entry within a `app.toml` file.
 - A command that is valid when executed using the Bash 3+ shell.
 
-### develop.toml (TOML)
+For each slice, buildpacks MUST specify zero or more path globs such that each path is either:
 
-```toml
-[[processes]]
-type = "<process type>"
-command = "<command>"
-```
+- Relative to the root of the app directory without traversing outside of the app directory.
+- Absolute, with `/` assumed to be the app directory.
```

Comment:
does this mean a path like `/foo` would resolve to `/workspace/app/foo`?

Reply:
It means `/foo`, `foo`, and `./foo` all resolve to `/workspace/app/foo`.

Reply:
is there a precedent for resolving `/` into the `/whatever` path? I'm not necessarily against it, but it could be awkward. Another option is to call `/` paths invalid and reject them.

Reply:
Addressed. We will only accept relative paths and (true) absolute paths that contain the app directory in the base path.